index.js

```diff
@@ -1,21 +1,29 @@
 const Sourcebit = require("./lib/sourcebit");
 
-module.exports.fetch = async (config, runtimeParameters) => {
+module.exports.fetch = (config, runtimeParameters, transformCallback) => {
   if (!config) {
     throw new Error(
       "ERROR: Could not find a valid `sourcebit.js` configuration file."
     );
   }
 
-  const instance = new Sourcebit({ runtimeParameters });
+  if (typeof runtimeParameters === "function") {
```

Comment:
This is in case the method is called without `runtimeParameters`, in which case the callback would be the second argument.

Reply:
Do you mean throwing an error if `runParameters.watch === true && typeof transformCallback !== 'function'`?